image/image.go

```diff
@@ -133,7 +133,7 @@ func InitializeData(imageID string) ([]*Layer, []*filetree.FileTree) {
 	var trees []*filetree.FileTree = make([]*filetree.FileTree, 0)
 
 	// save this image to disk temporarily to get the content info
-	// fmt.Println("Fetching image...")
+	fmt.Println("Fetching image...")
```

Comment:
Did you mean to uncomment this print statement?

Reply:
actually, I never meant to comment it out to begin with :)